docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -101,7 +101,8 @@ zeebe:
 ```
 
 The bucket specified with `bucketName` **must already exist**, Zeebe will not try to create one for you.
-To prevent misconfiguration, Zeebe will check at startup that the specified bucket exists and can be accessed.
+To prevent misconfiguration, Zeebe will check at startup that the specified bucket exists and can be accessed, but
+it will not prevent Zeebe to start up.
```

Comment:
```suggestion
To prevent misconfiguration, Zeebe will check at startup that the specified bucket exists and can be accessed, and log at WARN level if the bucket does not exist. 
```